jsonschema-core.xml

```diff
@@ -3291,11 +3296,11 @@ https://example.com/schemas/common#/$defs/count/minimum
             &RFC8259;
             &ldp;
             <reference anchor="ecma262"
-            target="http://www.ecma-international.org/publications/files/ECMA-ST/Ecma-262.pdf">
+            target="https://www.ecma-international.org/ecma-262/11.0">
```

Comment:
I support using ECMA-262 11th edition, but the PR title should change.

Their official page at https://www.ecma-international.org/publications/standards/Ecma-262.htm points to the URL, "https://www.ecma-international.org/ecma-262/11.0/index.html#title".